in/in_command.go

```diff
@@ -368,6 +369,20 @@ func (c InCommand) downloadFiles(
 		return err
 	}
 
+	if unpack {
+		for _, destinationPath := range files {
```

Comment:
What happens if the downloaded files are a mix of archives and non-archives? It looks like it will error, though I'd expect this use-case to be supported.